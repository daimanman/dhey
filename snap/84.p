{"C":1,"Duration":37897924,"EndTime":1537240578962428100,"Err":"","FileId":"3","InputFileName":"file","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537240578924530200,"Status":1,"TaskId":84,"Type":"FILE","Url":"http://192.168.1.192:9091/lhcenter/api/v1/oss","Z":""}