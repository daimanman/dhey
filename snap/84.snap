{"AvgTotal":0.0029921,"Fastest":0.0029921,"Slowest":0.0029921,"Average":0.0029921,"Rps":334.21342869556497,"AvgConn":0,"AvgDNS":0,"AvgReq":0,"AvgRes":0,"AvgDelay":0.0019944,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0019944,"DelayMin":0.0019944,"Lats":[0.0029921],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.0019944],"Total":2992100,"ErrorDist":{},"StatusCodeDist":{"404":1},"SizeTotal":19,"SizeReq":19,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0029921,"Count":1,"Frequentcy":1},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0},{"Mark":0.0029921,"Count":0,"Frequentcy":0}],"TaskId":84,"Duration":0.0029921}